packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -19,48 +17,33 @@ const resolveStats = require('./groups/resolveStats');
 const resolveOutput = require('./groups/resolveOutput');
 const basicResolver = require('./groups/basicResolver');
 const resolveAdvanced = require('./groups/resolveAdvanced');
+const { toKebabCase } = require('./utils/helpers');
 
-class WebpackCLI extends GroupHelper {
+class WebpackCLI {
     constructor() {
-        super();
-        this.groupMap = new Map();
         this.compilerConfiguration = {};
         this.outputConfiguration = {};
     }
-    setMappedGroups(args, inlineOptions) {
-        Object.keys(args).forEach((key) => {
-            this.setGroupMap(toKebabCase(key), args[key], inlineOptions);
-        });
-    }
-    setGroupMap(key, val, inlineOptions) {
-        if (val === undefined) return;
-        const opt = inlineOptions.find((opt) => opt.name === key);
-        const groupName = opt.group;
-        if (this.groupMap.has(groupName)) {
-            const pushToMap = this.groupMap.get(groupName);
-            pushToMap.push({ [opt.name]: val });
-        } else {
-            this.groupMap.set(groupName, [{ [opt.name]: val }]);
-        }
-    }
 
     /**
      * Responsible for handling flags coming from webpack/webpack
      * @private\
      * @returns {void}
      */
     _handleCoreFlags(parsedArgs) {
-        if (this.groupMap.has('core')) {
-            const coreFlags = this.groupMap.get('core');
-
-            // convert all the flags from map to single object
-            const coreConfig = coreFlags.reduce((allFlag, curFlag) => ({ ...allFlag, ...curFlag }), {});
-            const coreCliHelper = require('webpack').cli;
-            const coreCliArgs = coreCliHelper.getArguments();
-            // Merge the core flag config with the compilerConfiguration
-            coreCliHelper.processArguments(coreCliArgs, this.compilerConfiguration, coreConfig);
-            // Assign some defaults to core flags
-        }
+        const coreConfig = Object.keys(parsedArgs)
+            .filter((arg) => {
+                return coreFlagMap.has(toKebabCase(arg));
+            })
+            .reduce((acc, cur) => {
+                acc[toKebabCase(cur)] = parsedArgs[cur];
+                return acc;
+            }, {});
+        const coreCliHelper = require('webpack').cli;
```

Comment:
What about if we have webpack@4? We don't have `.cli`

Reply:
/cc @anshumanv

Reply:
oops, I think it worked, lemme recheck

Reply:
Yep, let's check it, but ci success so I think it should be work

Reply:
strange, passed in CI but fails locally 😞

Reply:
sending a follow up - sorry